DQM/SiStripMonitorClient/test/SiStripDQM_OfflineTkMap_Template_cfg_DB.py

```diff
@@ -112,9 +112,9 @@
     cms.PSet(mapName=cms.untracked.string('NumberOfOfffTrackCluster'),TopModules=cms.untracked.bool(True)),
     cms.PSet(mapName=cms.untracked.string('NumberOfOfffTrackCluster'),mapSuffix=cms.untracked.string("_autoscale"),mapMax=cms.untracked.double(-1.)),
     cms.PSet(mapName=cms.untracked.string('NumberOfOnTrackCluster')),
-    cms.PSet(mapName=cms.untracked.string('StoNCorrOnTrack')),
+    cms.PSet(mapName=cms.untracked.string('StoNCorrOnTrack',TopModules=cms.untracked.bool(True)),
```

Comment:
auto merge job (80X changes in to 81X) failed to resolve the above? Is not there a missing ')' after 'StoNCorrOnTrack'? In 81X we have https://github.com/cms-sw/cmssw/commit/0249733705cf5550174cf84e78c5accf5a88623c#diff-da712dd5a8dd05d67efa0e78be33b175